clrdefinitions.cmake

```diff
@@ -131,8 +131,10 @@ add_definitions(-DFEATURE_MANAGED_ETW_CHANNELS)
 add_definitions(-DFEATURE_MAIN_CLR_MODULE_USES_CORE_NAME)
 add_definitions(-DFEATURE_MERGE_CULTURE_SUPPORT_AND_ENGINE)
 if(WIN32)
-# Disable the following for UNIX altjit on Windows
-add_definitions(-DFEATURE_MERGE_JIT_AND_ENGINE)
+  if(FEATURE_MERGE_JIT_AND_ENGINE)
```

Comment:
I wonder why do we have this feature defined at so many places (I know it is not your change). All of them should inherit the definition from here - after removing the if (WIN32) here. 
Can you try to remove the other definitions and this "if" around the definition here?